### DISCUSSION TITLE
Can we reduce the quality of recording ?

### DISCUSSION THREAD

**Discussion by Greatz08** [mediar-ai/screenpipe#407] [General]
Recording Video At highest quality can easily fill remaining space of device so is it possible to either record at low quality which wont generate too big file but will be effective at same time in generating transcript for  our daily task ? Or can it be implemented in such a way that after it records for example 3 or 5  min video then it split that video in such a way that specific part is splited into mp4 file from which  transcript of our task is generated and whatever else this project create backup and then remove that video from storage and at the same time keep recording other task which user perform and do same thing again and again so that their wont be too big video unnecessarily and at the same time it will have all the record and ss of the task we had done whole day

**Comment by louis030195** [comment]
@Greatz08 not yet, how much storage do you have? how much storage ideally would you want screenpipe to take at most on a monthly basis?

**Reply by Greatz08** [reply]
@louis030195 I have less than 100 GB space available right now because i have many selfhosted services running with docker so bigger docker images consuming alot of space plus some of my files (i dont have any videos/movies as big files nor any heavy games) so yeah because of this i have some storage issues  so thought to ask if it is possible to be able record at low but llm understandable quality or maybe any auto compression mechanics might help or the idea i gave you to record for specific duration and convert it to knowledge base and split remove that part and keep recording further activities so that storage wont fill up plus knowledge base will have all our activities information to answer questions based on our daily activity

**Comment by louis030195** [comment]
hey @Greatz08

closing this as part of our cleanup - screenpipe v2.2.0 is now out with major improvements!

check it out: https://x.com/louis030195/status/2014403576475615312

if this is still relevant or you have questions, feel free to reopen this discussion. thanks for being part of the community!
